webrender/src/device/query_gl.rs

```diff
@@ -263,33 +267,40 @@ impl<T: NamedTag> GpuProfiler<T> {
     }
 
     pub fn start_marker(&mut self, label: &str) -> GpuMarker {
-        GpuMarker::new(&self.gl, label)
+        GpuMarker::new(&self.gl, label, self.ext_debug_marker)
     }
 
     pub fn place_marker(&mut self, label: &str) {
-        GpuMarker::fire(&self.gl, label)
+        GpuMarker::fire(&self.gl, label, self.ext_debug_marker)
     }
 }
 
 #[must_use]
 pub struct GpuMarker {
     gl: Rc<gl::Gl>,
+    ext_debug_marker: bool
```

Comment:
The whole struct is rather useless without this extension. If the users can't make a decision not to use it at all, we might just make it `gl: Option<Rc<gl::Gl>>` here instead of adding a flag